test/repl.jl

```diff
@@ -733,4 +733,14 @@ end
     end
 end
 
+@testset "JuliaLang/julia #55850" begin
+    tmp_55850 = mktempdir()
+    tmp_sym_link = joinpath(tmp_55850, "sym")
+    symlink(tmp_55850, tmp_sym_link; dir_target=true)
+    withenv("JULIA_DEPOT_PATH" => tmp_sym_link, "JULIA_LOAD_PATH" => nothing) do
```

Comment:
Without a trailing ":" the shipped stdlib caches aren't reachable so this test will needlessly precompile Pkg etc.
```suggestion
    withenv("JULIA_DEPOT_PATH" => tmp_sym_link * ":", "JULIA_LOAD_PATH" => nothing) do
```

Reply:
Should I also fix it in the julia PR (and the test where I borrowed this code from)?

Reply:
Yeah needed there too I believe